app/qml/ProjectDelegateItem.qml

```diff
@@ -160,4 +165,15 @@ Rectangle {
             anchors.bottom: parent.bottom
         }
     }
+
+    // Additional item
+    DelegateButton {
+      visible: itemContainer.isAdditional
```

Comment:
Is it not possible to just stick a button on the end of the listview? Clicking on the button would fetch new entries and fill a model with them. We would then not need a new project status nor any other reserved projects name.

Reply:
Of course, there are multiple ways how to implement such button. Some of the ideas were discussed with Martin and we agreed on introducing an extra item which seems to be the easiest and fastest solution.

Reply:
Well, for me it seems as two different things and joining them can make some troubles in future (like checking on many places if the project in list is actually a project not some other button). But if you guys are ok with that, I am too :))